components/library/lasso/Makefile

```diff
@@ -29,6 +29,7 @@ include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=	lasso
 COMPONENT_VERSION=	2.6.0
+COMPONENT_REVISION=	1
```

Comment:
I am sorry for the breakage. Can you add `REQUIRED_PACKAGES += developer/check` on top of `# Auto-generated dependencies`? I wasn't able to identify this component as using `check` when I updated check to 0.13.0.